linkchecker

```diff
@@ -542,7 +542,7 @@ for entry in config["authentication"]:
         attrs["strpattern"] = attrs["pattern"].pattern
         msg = _("Enter LinkChecker password for user %(user)s" \
                 " at %(strpattern)s:") % attrs
-        entry["password"] = getpass.getpass(console.encode(msg))
+        entry["password"] = getpass.getpass(msg)
```

Comment:
shouldn't we do explicit decoding here, since it's a boundary?

Reply:
getpass.getpass() works in Unicode:

```
mg@blynas: ~ $ python3
Python 3.8.2 (default, Mar 13 2020, 10:14:16) 
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import getpass
>>> getpass.getpass('Slaptažodis: ')
Slaptažodis: 
'ąčę'
```